devel/git-crypt/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+github.setup            AGWA git-crypt 0.6.0
+homepage                https://www.agwa.name/projects/git-crypt/
+categories              devel
+platforms               darwin
+maintainers             @nareshov
+license                 GPL-3
+checksums               rmd160  b151b4426cdf2c317f9ac15125ffbcb265511ba2 \
+                        sha256  d033017252a0bbec86dd67c2489eb59159b9c61856afc0502304fc32ded7bb80
+
+description             {Transparent file encryption in git.}
+long_description        git-crypt enables transparent encryption and \
+                        decryption of files in a git repository. Files which \
+                        you choose to protect are encrypted when committed, \
+                        and decrypted when checked out. git-crypt lets you \
+                        freely share a repository containing a mix of public \
+                        and private content. git-crypt gracefully degrades, \
+                        so developers without the secret key can still clone \
+                        and commit to a repository with encrypted files. \
+                        This lets you store your secret material (such as \
+                        keys or passwords) in the same repository as your \
+                        code, without requiring you to lock down your entire \
+                        repository.
+
+depends_lib-append      path:lib/libssl.dylib:openssl
+depends_build-append    bin:xsltproc:libxslt
+
+use_configure           no
+destroot.args           ENABLE_MAN=yes PREFIX=${prefix}
+build {}
```

Comment:
Why did you disable the build phase? The [git-crypt Makefile](https://github.com/AGWA/git-crypt/blob/0.6.0/Makefile#L36) supports our default build.target (`all`).

Reply:
I'm removing this line which disables the build phase and updating the PR.
Thank you for all the helpful feedback for a newcomer such as me!